src/test/java/amazin/BaseTest.java

```diff
@@ -29,61 +30,57 @@ public static void tearDownAfterClass() throws Exception {
     @Before
     public void setUp() throws Exception {
         book = new Book(DEFAULT_ID, DEFAULT_NAME, DEFAULT_DESCRIPTION, DEFAULT_ISBN, DEFAULT_PICTURE, DEFAULT_AUTHOR,
-                DEFAULT_PUBLISHER, DEFAULT_SKU, DEFAULT_INVENTORY, DEFAULT_PRICE);
+                DEFAULT_PUBLISHER, DEFAULT_INVENTORY, DEFAULT_PRICE);
     }
 
     @After
     public void tearDown() throws Exception {
     }
 
     protected void testAllFieldsDefaultExcept(Fields.Book field) {
-        for(Fields.Book f : Fields.Book.values()) {
+        for (Fields.Book f : Fields.Book.values()) {
             if (f.compareTo(field) != 0)
                 assertFieldEqualsDefault(f);
         }
     }
 
     protected void testSetField(Fields.Book field, Object val) {
-        switch(field) {
-            case ID:
-                book.setId((Long) val);
-                break;
-
-            case NAME:
-                book.setName((String) val);
-                break;
-
-            case AUTHOR:
-                book.setAuthor((String) val);
-                break;
-
-            case DESCRIPTION:
-                book.setDescription((String) val);
-                break;
-
-            case ISBN:
-                book.setISBN((Integer) val);
-                break;
-
-            case PICTURE:
-                book.setPicture((String) val);
-                break;
-
-            case PUBLISHER:
-                book.setPublisher((String) val);
-                break;
-
-            case SKU:
-                book.setSKU((Integer) val);
-                break;
-
-            case INVENTORY:
-                book.setInventory((Integer) val);
-                break;
-
-            case PRICE:
-                book.setPrice((Double) val);
-                break;
+        switch (field) {
```

Comment:
Please indent correctly and not add unnecessary commits

Reply:
This is the linting plugin.